README.md

```diff
@@ -467,6 +467,16 @@ Manages the GPG keys that Apt uses to authenticate packages.
 
 This module is tested and officially supported on Debian 6 and 7 and Ubuntu 10.04, 12.04, and 14.04. Testing on other platforms has been light and cannot be guaranteed.
 
+This module is not designed to be split across stages. If you want to ensure that apt update is run before all packages, add a proper dependency like this:
+
+~~~puppet
+Class['apt::update'] -> Package<||>
```

Comment:
I mean,
1. if you are on a Debian-like OS, by default, Puppet will use *apt* to manage packages.
2. if you specify *provider=apt*, it will also use *apt* to manage packages.

Once you use:
```
Package <| provider == 'apt' |> 
```
, will it realize all resources or only the 2nd case resources?